OpenEdXMobile/src/main/java/org/edx/mobile/view/adapters/CourseUnitPagerAdapter.java

```diff
@@ -57,30 +57,44 @@ public static boolean isCourseUnitVideo(CourseComponent unit) {
 
     @Override
     public Fragment getItem(int pos) {
-        CourseComponent unit = getUnit(pos);
+        final CourseComponent unit = getUnit(pos);
+        final CourseComponent minifiedUnit;
+        {
+            // FIXME: Remove this code once LEARNER-6713 is merged
```

Comment:
Move this line under this block for clarity.
`final CourseComponent unit = getUnit(pos);`